apps/demo-app/package.json

```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0-beta.15",
   "type": "module",
   "scripts": {
-    "start": "pnpm run dev",
-    "dev": "cd ../../packages/sit-onyx && pnpm build && cd ../../apps/demo-app && vite",
+    "dev": "turbo watch vite --filter demo-app",
```

Comment:
do you know why this might fail? proxyfire?
<img width="868" alt="image" src="https://github.com/user-attachments/assets/98213095-93c7-485b-aa97-17c70834572e">

but would that mean we're not able to start the local server when being offline anymore? :O

Reply:
proxyfire shows no errors. 
same issue happens on the playground

Reply:
How can I reproduce this issue?

Reply:
```
➜  onyx git:(fix/storybook-deployment) cd apps/demo-app 
➜  demo-app git:(fix/storybook-deployment) pnpm run dev
```

![image](https://github.com/user-attachments/assets/b7048348-e963-4572-9083-8c4e12d2fc30)